File: cron-job/azure-function/score.py
import datetime


class Score:
    def __init__(
        self,
        username: str,
        completion_date: datetime.date,
        time_in_seconds: int,
    ):
        self.username = username
        self.completion_date = completion_date
        self.time_in_seconds = time_in_seconds

File: NytStrawHatLeaderboard/NytStrawHatLeaderboard.Data/Entities/Score.cs
using System;
using System.Collections.Generic;

namespace NytStrawHatLeaderboard.Data.Entities;

public partial class Score
{
    public string Username { get; set; } = null!;

    public DateOnly CompletionDate { get; set; }

    public short TimeInSeconds { get; set; }

    public virtual Player UsernameNavigation { get; set; } = null!;
}
